src/main/assets/scss/main.scss

```
@@ -44,3 +44,59 @@ section {
 .red-text {
   color: govuk-colour('red');
 }
+
+.hmcts-progress-bar__icon {
+  position: relative;
+  background-color: govuk-colour('white');
+  border: 4px solid govuk-colour('mid-grey');
+  border-radius: 50%;
+  box-sizing: border-box;
+  display: block;
+  height: 44px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 44px; }
+
+.hmcts-progress-bar__icon--complete {
+  background-color: govuk-colour('blue');
+  border: 4px solid govuk-colour('blue');
+  background-image: url(/hmcts-assets/images/icon-progress-tick.svg);
+  background-position: 50% 50%;
+  background-repeat: no-repeat; }
+
```

Adding background-size makes the tick icon slightly bigger to match the Figma design:
```suggestion
  background-repeat: no-repeat;
  background-size: 23px;
}

```